package.json

```diff
@@ -10,19 +10,26 @@
   "umd:main": "dist/redux-on-action.umd.js",
   "scripts": {
     "build": "microbundle src/index.js -o dist/redux-on-action.js",
-    "test": "eslint src",
-    "release": "npm run build && npm t && git commit -am $npm_package_version && git tag $npm_package_version && git push && git push --tags && npm publish"
+    "lint": "eslint src",
+    "test": "jest",
+    "release": "npm run lint && npm run build && npm t && git commit -am $npm_package_version && git tag $npm_package_version && git push && git push --tags && npm publish"
   },
   "devDependencies": {
+    "@babel/core": "7.1.2",
+    "@babel/preset-env": "7.1.0",
+    "babel-core": "7.0.0-bridge.0",
+    "babel-jest": "23.6.0",
```

Comment:
Why is babel necessary?

Reply:
Babel is necessary, because Jest runs on Node JS environment; so we need to transpile the test codes into what can be run on Jest